inst/extdata/nonint.do

```diff
@@ -0,0 +1,12 @@
+clear all
+
+set obs 2
+
+gen double v1 = _n
```

Comment:
maybe we should add a note, why nonint.factor labels may look strange if created from float values? float is still Statas default type.